src/libraries/Microsoft.PowerFx.Core/Public/Values/NamedValue.cs

```diff
@@ -25,7 +44,19 @@ public NamedValue(KeyValuePair<string, FormulaValue> pair)
         public NamedValue(string name, FormulaValue value)
```

Comment:
>value [](http://example.com/codeflow?start=52&length=5)

Now that this is being used, consider throwing an ArgumentNullException for this argument.

Reply:
updated.